PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetHdEdxCorrelations.cxx

```diff
@@ -1495,6 +1501,45 @@ namespace PWGJE
       return kTRUE;
     }
 
+  Double_t AliAnalysisTaskEmcalJetHdEdxCorrelations::GetFlattenedEPAngle(Double_t uncorrectedAngle){
+     if (!gGrid)
+      {
+        TGrid::Connect("alien://");
+      }
+        // Open the ROOT file in read mode
+      TFile file("alien:///alice/cern.ch/user/p/psteffan/epCorrections.root", "READ");
+
+      // Read the TTree from the ROOT file
+      TTree *tree = dynamic_cast<TTree *>(file.Get("V0C"));
```

Comment:
It appears that this will be opened in every job and read every event. If true, that's incredibly expensive, and will cause huge load on the grid, so it needs to be fixed.

The file doesn't appear to depend on the event, so it should be opened during the AddTask phase and are stored in variables which are streamed. If it for some reason needs to be opened separately job-by-job, then it should be in `UserCreateOutputObjects` or `UserExecOnce`